radicle/src/storage.rs

```diff
@@ -174,13 +176,17 @@ pub struct Remote<V> {
     pub id: PublicKey,
     /// Git references published under this remote, and their hashes.
     pub refs: SignedRefs<V>,
+    // TODO(finto): Why does this live here? They could be a delegate of one project and not of another?
```

Comment:
I'm curious about this :eyes:

Reply:
This basically says that this remote belongs to a delegate of the project, ie. the public key is part of the project delegation. Does that make sense?

Reply:
Ah, I think the comment threw me off with the use of "a project", but this `Remote` struct is just for a single project?

Reply:
Yeah, it's just for a single project. The refs it contains are scoped to a project/repo.

Reply:
Cool! I updated the doc string :)